### PR TITLE
Support DebugFission with `cc_configure`

### PR DESCRIPTION
**Pull request by GregBowyer**
This is a basic PR to hopefully try to support DebugFission with toolchains / CROSSTOOLS that are autogenerated from the `cc_configure` rules.

I do not know if there is additional work to be done here, I am guessing it might be good to not only attempt the flag but also ensure linkage happens.

### CODE REVIEW

**Comment by cbsmith** [comment]
This looks purty

**Comment by buchgr** [comment]
fyi @mhlopko is OOO for the next two weeks.

**Comment by GregBowyer** [comment]
@mhlopko any chance for a review?

**Comment by mhlopko** [comment]
Hi Greg, 

I'm sorry you had to wait this long. I'm back from the vacation and I'll take a look next week. I want to double check how gcc/clang/clang on mac behave so we detect fission properly, and I want to rediscover what 'is_using_fission' crosstool variable is for and investigate opportunities for cleanup. Stay tuned :)

**Comment by GregBowyer** [comment]
Should I rebase this?

**Comment by jin** [comment]
What's the status of this? @scentini?

File: tools/cpp/unix_cc_configure.bzl [hunk 1]
```diff
@@ -189,6 +189,17 @@ def _is_gold_supported(repository_ctx, cc):
     ])
     return result.return_code == 0
 
+def _is_fission_supported(repository_ctx, cc):
+    """Checks if `DebugFission` is supported by the C compiler."""
+    result = repository_ctx.execute([
+        cc,
+        "-gsplit-dwarf",
+        "-g",
+        "-c",
```
Comment (ulfjack) [inline tools/cpp/unix_cc_configure.bzl]: You probably need to add -o /dev/null to avoid a non-sensical output file in the tools tree.

**Comment by aiuto** [comment]
@GregBowyer This is very old and would need work to update. Is it still valid?

**Comment by GregBowyer** [comment]
Doubtful lets close it out and redo it given all the changes to cc rules

**Pull request closed without merging**